go.mod

```diff
@@ -154,4 +155,6 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
-go 1.19
+replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-display v1.30.0-hex-display
```

Comment:
Can you please add this comment above that we also have in other projects, so we know why we need this:

```
// We want to format raw bytes as hex instead of base64. The forked version
// allows us to specify that as an option.
```

Reply:
Added.